tests/FluentTest.php

```diff
@@ -913,12 +913,12 @@ public function testWriteWithLocale()
         $item2->Title = 'English 2';
         $item2->write();
 
-        // check alternate locale title unchanged
+        // check alternate locale title reflects the default locale value
         $es2Title = Fluent::with_locale('es_ES', function () use ($item2ID) {
             $item2 = FluentTest_TranslatedObject::get()->byId($item2ID);
             return $item2->Title;
         });
-        $this->assertEquals($es2Title, 'Spanish 2');
+        $this->assertEquals($es2Title, 'English 2');
```

Comment:
Hm, this raises an interesting question.

Create record in spanish locale, save.
Go to english locale, modify.
Going back to spanish locale, and this version has reverted to the english locale (because it was null and thus override it).

You agree @robbieaverill that this is bad behaviour right? Is there a way we can prevent this?